python/tvm/relay/frontend/tflite.py

```diff
@@ -1422,10 +1423,48 @@ def convert_pad(self, op):
         # convert list of lists to tuple of tuples
         paddings = tuple(tuple(l) for l in pad_list)
 
-        # Use default pad_value 0 because TFLite does not support constant_values parameter
+        # Use default pad_value 0 because TFLite PAD does not support constant_values parameter
         out = _op.nn.pad(in_expr, paddings)
         return out
 
+    def convert_mirror_pad(self, op):
+        """Convert TFLite MIRROR_PAD"""
+        try:
+            from tflite.Operator import Operator
+            from tflite.BuiltinOptions import BuiltinOptions
+            from tflite.MirrorPadOptions import MirrorPadOptions
+        except ImportError:
+            raise ImportError("The tflite package must be installed")
+
+        # the quantized form MirrorPad is not yet implemented in TFLite.
+        if self.is_quantized(op):
+            raise tvm.error.OpNotImplemented(
+                'TFlite quantized MIRROR_PAD operator is not supported yet.')
```

Comment:
Pedantry : A not implemented error suggests to me as a user that the support exists in the tflite tooling but the TVM stack has not yet implemented the support i.e. the tflite tooling can't yet produce the mirror_pad operation, do we need a different notification here actually asking the user to report an issue asking for a feature to be added ?

Reply:
I think this exception will not be triggered when quantized MIRROR_PAD is not implemented by TFLite. If the exception is triggered, it certainly means the TVM stack doesn't support quantized MIRROR_PAD.